modules/axhal/src/arch/aarch64/hv/exception.S

```diff
@@ -97,13 +96,13 @@ exception_vector_base_el2:
 
     // current EL, with SP_ELx
     INVALID_EXCP_EL2 0 1
-    HANDLE_CURRENT_IRQ
+    HANDLE_IRQ
     INVALID_EXCP_EL2 2 1
     INVALID_EXCP_EL2 3 1
 
     // lower EL, aarch64
     HANDLE_LOWER_SYNC
-    HANDLE_LOWER_IRQ
+    INVALID_EXCP_EL2 2 1
```

Comment:
For now, irq from "current el" irq and "lower el" can be handled by a same routine

like this: https://github.com/tsemo4917/rtshyper/blob/297c0f24262f2f6ac8626b00a0c884bf301640fb/src/arch/aarch64/exception.rs#L177

Reply:
I have configured ArceOS to handle its IRQs within axhal. If an IRQ is not found, meaning it does not belong to ArceOS, the process will jump to the `LOWER_AARCH64_IRQ_HANDLER`, which is registered in AXVM.

Reply:
emmmmmm

Reply:
since ArceOS runs in EL2, how does it handle its IRQs within axhal? 

All exceptions happened in EL2 willl be routed to this exception vector!